packages/rspack-test-tools/tests/configCases/postcss-loader/with-previous-source-map/rspack.config.js

```diff
@@ -18,7 +18,11 @@ module.exports = {
 						}
 					},
 					{
-						loader: "sass-loader"
+						loader: "sass-loader",
+						options: {
+							// use legacy API to generate source maps
```

Comment:
I added `api: 'legacy'` here because the modern Sass API disabled [sourceMapIncludeSources: true](https://sass-lang.com/documentation/js-api/interfaces/options/#sourceMapIncludeSources) by default, see https://github.com/webpack-contrib/sass-loader/issues/1222